lib/auth.coffee

```diff
@@ -23,6 +23,15 @@ getAuth = (deps, opts) ->
 	twoFactor = require('./2fa')(deps, opts)
 	exports = {}
 
+	normalizeError = (err) ->		#todo
```

Comment:
Can we make this name a tiny bit more specific? It's not clear from this exactly what this does. `normalizeAuthError` would be a good start I think, but suggestions welcome.